src/import/FSHImporter.ts

```diff
@@ -2426,7 +2480,8 @@ export class FSHImporter extends FSHVisitor {
       };
       if (
         !(pc.containsPathContext(ctx) || pc.containsCodePathContext(ctx)) &&
-        location.startColumn - DEFAULT_START_COLUMN > 0
+        location.startColumn - DEFAULT_START_COLUMN > 0 &&
+        !suppressError
```

Comment:
This is a little nit-picky, but `!suppressError` is the easiest operation of this boolean logic -- and there's no need to check anything else if we're suppressing errors -- so I'd suggest this should be _first_ in the chain of boolean logic so it can short-circuit when suppressing errors.